pkg/agent/types/annotations.go

```diff
@@ -23,4 +23,7 @@ const (
 
 	// NodeWireGuardPublicAnnotationKey represents the key of the Node's WireGuard public key in the Annotations of the Node.
 	NodeWireGuardPublicAnnotationKey string = "node.antrea.io/wireguard-public-key"
+
+	// ExternalIPPoolAnnotationKey is the key of the Service annotation that specifies the Service's desired external IP pool.
+	ExternalIPPoolAnnotationKey string = "antrea.io/external-ip-pool"
```

Comment:
Per offline discussion with @tnqn , probably we name the key: "service.antrea.io/external-ip-pools". "pools" is for future support of v4 and v6 pools.